src/rely/reporters/JunitReporter.re

```diff
@@ -0,0 +1,63 @@
+let testResultToTestCase = (testResult: Reporter.testResult) => {
+  let time =
+    switch (testResult.duration) {
+    | Some(Seconds(s)) => s
+    | None => 0.
+    };
+
+  switch (testResult.testStatus) {
+  | Passed =>
+    Junit.Testcase.pass(
+      ~name=testResult.title,
+      ~classname=testResult.fullName,
+      ~time,
+    )
+  | Skipped =>
+    Junit.Testcase.skipped(
+      ~name=testResult.title,
+      ~classname=testResult.fullName,
+      ~time,
+    )
+  | Failed(description, _location_opt, message) =>
```

Comment:
The first argument to this constructor is actually the failure message from the matcher and the last one is the stack trace. I think we might want to update what we pass here accordingly?

Reply:
Yes! Thank you, I just stuck something in there while hacking this together and wanted to ask you about it.